src/mlpack/core/tree/cover_tree/dual_tree_traverser.hpp

```diff
@@ -87,19 +87,23 @@ class CoverTree<MetricType, StatisticType, MatType, RootPointPolicy>::
    * Helper function for traversal of the two trees.
    */
   void Traverse(CoverTree& queryNode,
-                std::map<int, std::vector<DualCoverTreeMapEntry> >&
-                    referenceMap);
+                std::map<int,
+                         std::vector<DualCoverTreeMapEntry>,
+                         std::greater<int>>& referenceMap);
 
   //! Prepare map for recursion.
   void PruneMap(CoverTree& queryNode,
-                std::map<int, std::vector<DualCoverTreeMapEntry> >&
-                    referenceMap,
-                std::map<int, std::vector<DualCoverTreeMapEntry> >&
-                    childMap);
+                std::map<int,
+                         std::vector<DualCoverTreeMapEntry>,
+                         std::greater<int>>& referenceMap,
+                std::map<int,
+                         std::vector<DualCoverTreeMapEntry>,
+                         std::greater<int>>& childMap);
 
   void ReferenceRecursion(CoverTree& queryNode,
-                          std::map<int, std::vector<DualCoverTreeMapEntry> >&
-                              referenceMap);
+                          std::map<int,
+                                   std::vector<DualCoverTreeMapEntry>,
+                                   std::greater<int>>& referenceMap);
```

Comment:
I guess, at some places we are not really consistent with the indentation, because sometimes the parameter aligns with the first line, sometimes it's tabbed twice. For example `dual_tree_traverser_impl.hpp` uses the tabbed twice style, which is what the style guide recommends, so to be consistent let's adapt the style here?

Reply:
I will fix the indentation. :+1: